app/controllers/report_controller.rb

```diff
@@ -72,8 +72,11 @@ def upload
     end
     if params[:upload] && params[:upload][:file] && params[:upload][:file].respond_to?(:read)
       @sb[:overwrite] = !params[:overwrite].nil?
+      @sb[:preserve_owner] = !params[:preserve_owner].nil?
```

Comment:
`@sb[:preserve_owner] = params[:preserve_owner].present?` sounds better :)

Reply:
ok, I will change
it would look more easy to read, but will dissonance with the line above it...

Reply:
I'll take care of that in https://github.com/ManageIQ/manageiq-ui-classic/pull/5094

Reply:
@ZitaNemeckova  Ohh, I see, you have change that in  PR